libnetwork/internal/util/parse.go

```diff
@@ -34,3 +34,20 @@ func ParseVlan(vlan string) (int, error) {
 	}
 	return v, nil
 }
+
+// ParseIsolate parses the isolate option
+func ParseIsolate(isolate string) (string, error) {
```

Comment:
Can we transform the logic into a switch case?
```golang
switch isolate {
case "":
  return "false", nil
case "strict":
  return isolate, nil
default:
   // parse bool
}
```

I'd find it easier to read and to extend.

Reply:
Sure. Looks good. Thank you.